### PR TITLE
fix(a11y): ensure id attributes are unique

### PR DESCRIPTION
**Pull request by mcoker**
fixes https://github.com/patternfly/patternfly-next/issues/616

### CODE REVIEW

File: src/patternfly/components/ChipGroup/examples/chip-group-select-collapsed-example.hbs [hunk 1]
```diff
@@ -1,23 +1,23 @@
-{{#> chip-group}}
+{{#> chip-group chip--id="select-collapsed"}}
```
Comment (christiemolloy) [inline src/patternfly/components/ChipGroup/examples/chip-group-select-collapsed-example.hbs]: should this be `chip-group--id="select-collapsed"` ?

File: src/patternfly/demos/PageLayout/examples/page-layout-expandable-nav-example.hbs [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {{#> background-image}}
 {{/background-image}}
 
-{{#> page page--attribute='id="page-layout-expandable-nav"'}}
+{{#> page page--id="page-layout-expandable-nav"}}
```
Comment (jgiardino) [inline src/patternfly/demos/PageLayout/examples/page-layout-expandable-nav-example.hbs]: This is the same issue as mentioned above:
`<div class="pf-l-page" page-layout-expandable-nav>`

Comment (mcoker) [reply src/patternfly/demos/PageLayout/examples/page-layout-expandable-nav-example.hbs]: d'oh!

File: src/patternfly/components/ChipGroup/examples/chip-group-select-collapsed-example.hbs (hunk 1 above)
Comment (mcoker) [reply src/patternfly/components/ChipGroup/examples/chip-group-select-collapsed-example.hbs]: good call, changed it to `chip-group--id`

**Comment by mcoker** [comment]
@christiemolloy @jgiardino updated the PR

File: src/patternfly/demos/PageLayout/examples/page-layout-default-nav-header-example.hbs [hunk 3]
```diff
@@ -1,6 +1,6 @@
 {{#> background-image}}
 {{/background-image}}
-{{#> page page--attribute='id="page-layout-default-nav"'}}
+{{#> page page--id="page-layout-default-nav"}}
```
[the remark this answers is not in the document]
Comment (jgiardino) [reply src/patternfly/demos/PageLayout/examples/page-layout-default-nav-header-example.hbs]: I'm seeing this in all the page demos so far, so I'll stop commenting in individual page demo example files.

[the remark this answers is not in the document]
Comment (mcoker) [reply src/patternfly/demos/PageLayout/examples/page-layout-default-nav-header-example.hbs]: Sorry about that, should be fixed now.

File: src/patternfly/demos/PageLayout/page-template-header-tools-elements.hbs [hunk 4]
```diff
@@ -16,11 +16,11 @@
         {{/button}}
       {{/toolbar-item}}
       {{#> toolbar-item toolbar-item--modifier="pf-u-sr-only pf-u-visible-on-md"}}
-        {{#> dropdown dropdown-toggle--modifier="pf-m-plain" id="dropdown-example-collapsed" dropdown--attribute="pf-m-plain" dropdown--HasToggleIcon="true"}}
-          Kyle Baker
+        {{#> dropdown id=(concat page--id '-dropdown') dropdown-toggle--modifier="pf-m-plain" dropdown--attribute="pf-m-plain" dropdown--HasToggleIcon="true"}}
+          <i class="fas fa-cog"></i>
```
Comment (jgiardino) [inline src/patternfly/demos/PageLayout/page-template-header-tools-elements.hbs]: When you removed "Kyle Baker" you also removed the only accessible text string associated with this component. Can you add an `aria-label` to this dropdown?

Also, at sizes larger than 992px, I'm seeing this which looks like a mistake. Do we need an issue for that? 
image

**Pull request merged**